.github/azure-pipelines.yaml

```diff
@@ -19,6 +19,7 @@ extends:
     COMPLEX: true
     GCC_CONFIG: config/defaults/config.LINUX_GFORTRAN.mk
     INTEL_CONFIG: config/defaults/config.LINUX_INTEL.mk
-    INPUT_FILES: true
-    STEPS_TEST_REAL: testflo -v . -n 1
-    STEPS_TEST_COMPLEX: export PETSC_ARCH=complex-opt-\$COMPILERS-\$PETSCVERSION && testflo -v . -m "cmplx_test*" -n 1
\ No newline at end of file
+    BUILD_REAL: .github/build_real.sh
```

Comment:
These can be set as default values in the template

Reply:
Yes, I did set the defaults in the template, I'll remove them from here.